rootfs-builder/rootfs.sh

```diff
@@ -189,7 +219,16 @@ copy_kernel_modules()
 	OK "Kernel modules copied"
 }
 
-OSBUILDER_VERSION="unknown"
+error_handler()
+{
+	[ "$?" -eq 0 ] && return
+
+	if [ -n "$GRACEFUL_EXIT" ] && [ -n "$BUILD_CAN_FAIL" ]; then
+		info "Detected a build error, marking build as failed and exiting"
```

Comment:
I would be slightly clearer if this message explained that the script is exiting "successfully" even though an error was detected. It's rather an awkward thing to explain succinctly, but maybe something like:

> Detected a build error but distro $DISTRO is permitted to fail, so exiting successfully.